vtysh/vtysh.c

```diff
@@ -79,8 +79,8 @@ struct vtysh_client vtysh_client[] = {
 	{.fd = -1, .name = "eigrpd", .flag = VTYSH_EIGRPD, .next = NULL},
 	{.fd = -1, .name = "babeld", .flag = VTYSH_BABELD, .next = NULL},
 	{.fd = -1, .name = "sharpd", .flag = VTYSH_SHARPD, .next = NULL},
-	{.fd = -1, .name = "watchfrr", .flag = VTYSH_WATCHFRR, .next = NULL},
 	{.fd = -1, .name = "pbrd", .flag = VTYSH_PBRD, .next = NULL},
+	{.fd = -1, .name = "watchfrr", .flag = VTYSH_WATCHFRR, .next = NULL},
```

Comment:
if watchfrr needs to be last let's add a code comment here to that effect

Reply:
has this comment been addressed?

Reply:
Quentin probably has not gotten back to this.  I've been giving him a bunch of other stuff to take care of :)

Reply:
It doesn't need to be last, I changed that because I misread it, it does not change anything either way afaict

Reply:
If this change isn't needed we should keep the file as is.

Reply:
Changed it back.